tests/Database/DatabaseEloquentBuilderTest.php

```diff
@@ -1049,6 +1050,26 @@ public function testOldestWithColumn()
         $builder->oldest('foo');
     }
 
+    public function testUpdate()
+    {
+        Carbon::setTestNow(
```

Comment:
You can just write `Carbon::setTestNow($now = '2017-10-10 10:10:10');` it uses `::parse` internally here.